docs/docs/guides/remote-machine-learning.md

```diff
@@ -11,6 +11,10 @@ To alleviate [performance issues on low-memory systems](/docs/FAQ.mdx#why-is-imm
 Smart Search and Face Detection will use this feature, but Facial Recognition is handled in the server.
 :::
 
+:::danger
+When using remote machine learning, the thumb images are sent to the remote machine learning, use this option carefully when running this on a public computer or a paid processing cloud.
```

Comment:
```suggestion
When using remote machine learning, the thumbnails are sent to the remote machine learning container. Use this option carefully when running this on a public computer or a paid processing cloud.
```
Generally I don't really like the sound of this and I'm not even sure we need it

Reply:
I'll remove it if it's not relevant, I do think it's worth letting users know that there is a possibility of sensitive information being leaked if they choose to use a company that provides cloud processing power

Reply:
But that's like internet 101. Of course it's not a good idea to send images around the world without any encryption. I think I'd be happy with only keeping the first part: "When using remote machine learning, the thumbnails are sent to the remote machine learning container."